src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1253,6 +1253,85 @@ bool Ledger::visitAccountItems (
     }
 }
 
+bool Ledger::getAccountObjects (Account const& account,
+    LedgerEntryType const type, uint256 dirIndex, uint256 const& entryIndex,
+    std::uint32_t const limit, Json::Value& jvResult) const
+{
+    auto const rootDirIndex = getOwnerDirIndex (account);
+    auto found = false;
+
+    if (dirIndex.isZero ())
```

Comment:
`base_uint::isZero` and `base_uint::isNonZero` are deprecated.

Recommend:
    if (dirIndex == zero)

and
    if (dirIndex != zero)